kernel/aster-nix/src/net/socket/vsock/stream/socket.rs

```diff
@@ -0,0 +1,379 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use core::sync::atomic::AtomicBool;
+
+use atomic::Ordering;
+
+use super::{connected::Connected, connecting::Connecting, init::Init, listen::Listen};
+use crate::{
+    events::IoEvents,
+    fs::{file_handle::FileLike, utils::StatusFlags},
+    net::socket::{
+        vsock::{addr::VsockSocketAddr, VSOCK_GLOBAL},
+        SendRecvFlags, SockShutdownCmd, Socket, SocketAddr,
+    },
+    prelude::*,
+    process::signal::{Pollee, Poller},
+};
+
+pub struct VsockStreamSocket {
+    status: RwLock<Status>,
+    is_nonblocking: AtomicBool,
+    pollee: Pollee,
```

Comment:
Since `Init`, `Listen` and `Connected` all have pollee, this pollee can be removed.